app/views/cases/confirm_destroy.html.slim

```diff
@@ -23,7 +23,7 @@
         = @case.subject
 
     p
-      = f.text_area :reason_for_deletion
+      = f.text_area :reason_for_deletion, @case.reason_for_deletion
```

Comment:
I don't think you need to pass the current value for the reason on to the method here, it should automatically populate the field even without this change :)

Reply:
you are right tried 👍